Upgrade rclone and base image

mgumz opened the pull request:
This PR bumps the packaged rclone version to v1.50.2. The changelog
for this release is available (here)[1]. The major changes (to
rclone-1.49.1) are addressing several memory leaks and more robustness.

Also part of this PR: 'coreutils' and a proper 'date' implementation are now part of the image. This allows to use ISO8601 formatted dates.

This PR also bumps the base image version to Alpine-3.11.3.

In addition, the Dockerfile now uses opencontainers.org labels to
annotate the created container image.

[1]: https://github.com/rclone/rclone/blob/master/docs/content/changelog.md#v1502---2019-11-19

mgumz committed:
Upgrade rclone and base image

This commit bumps the packaged rclone version to v1.50.2. The changelog
for this release is available (here)[1]. The major changes (to
rclone-1.49.1) are addressing several memory leaks and more robustness.

This commit also bumps the base image version to Alpine-3.11.3.

In addition, the Dockerfile now uses opencontainers.org labels to
annotate the created container image.

[1]: https://github.com/rclone/rclone/blob/master/docs/content/changelog.md#v1502---2019-11-19

hsychla commented on Dockerfile:
```diff
@@ -10,9 +12,25 @@ RUN cd rclone-*-linux-amd64 && \
+LABEL org.opencontainers.image.url="https://github.com/travelping/docker-rclone"
+LABEL org.opencontainers.image.source="https://github.com/travelping/docker-rclone"
+LABEL org.opencontainers.image.version=$VERSION
+LABEL org.opencontainers.image.vendor="travelping.com"
```
> **org.opencontainers.image.vendor** Name of the distributing entity, organization or individual.

Sounds more like "Travelping GmbH" to me

mgumz committed:
Add coreutils to get a proper 'date' implementation

A pure busybox 'date' implementation is not able to parse proper ISO8601
dates. Such a feature might be required in the scripts used in
combination with the 'inotify-tools'. Thus, this commits adds
'coreutils' in order to get a proper 'date' implementation.
